aws/table_aws_elasticache_replication_group.go

```diff
@@ -0,0 +1,229 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elasticache"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsElasticCacheReplicationGroup(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_elasticache_replication_group",
+		Description: "AWS ElastiCache Replication Group",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("replication_group_id"),
+			ShouldIgnoreError: isNotFoundError([]string{"ReplicationGroupNotFoundFault", "InvalidParameterValue"}),
+			Hydrate:           getElasticCacheReplicationGroup,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listElasticCacheReplicationGroups,
```

Comment:
Can we update the function names to use `ElastiCache` instead of using `ElasticCache`?
```suggestion
			Hydrate: listElastiCacheReplicationGroups,
```
```suggestion
			Hydrate: listElasticCacheReplicationGroups,
```